Player Guides (#5)
FabricMC/fabric-docs: players/installing-fabric.md

---
title: Installing Fabric
description: A step by step guide on how to install Fabric.
---

# Installing Fabric

> This guide will walk you through installing Fabric for the Minecraft Launcher.

Reviewer: ```suggestion
This guide will walk you through installing Fabric for the official Minecraft Launcher.
```

For third party launchers, you should consult their documentation.

## 1. Download the Fabric Installer

You can download the Fabric Installer from the [Fabric Website](https://fabricmc.net/use/).

> For windows, it's recommended you download the `.exe` version (`Download For Windows`), as it doesn't require java to be installed on your system (it will use the java installation that comes with the Minecraft Launcher).

Reviewer: ```suggestion
If you use Windows, download the `.exe` version (`Download For Windows`), because it doesn't require Java to be installed on your system. It instead uses Java that came with the official launcher.
```

> For macOS and Linux, you should download the `.jar` version, this may mean you need to install java firstly though.

Reviewer: ```suggestion
For macOS and Linux, you should download the `.jar` version. Sometimes, you need to install Java before this step.
```

## 2. Run the Fabric Installer

::: warning

> Before running the installer, you must close the Minecraft Launcher if it is running, along with any other Minecraft instances that are currently running.

Reviewer: ```suggestion
Close Minecraft and the Minecraft Launcher first before installing.
```

> Before running the installer, you must close the Minecraft Launcher if it is running, along with any other Minecraft instances that are currently running.

Reviewer 2: I think this is fine to keep, however the installer will show a warning if you try to install when the launcher is open

> **You must also ensure that you have played the version of Minecraft you want to use Fabric for at least once, otherwise the installer will not be able to find the version's files.**

Reviewer 2: Not true, please remove.

> **You must also ensure that you have played the version of Minecraft you want to use Fabric for at least once, otherwise the installer will not be able to find the version's files.**

Reviewer: ```suggestion
**If you have not played the version of Minecraft you want to use Fabric on before, play it once before installing. Otherwise, the installer cannot find the version's files.**
```

> **You must also ensure that you have played the version of Minecraft you want to install Fabric for at least once, otherwise the installer will not be able to find the version's files.**

Reviewer: ```suggestion
**Before installing, make sure to play the version of the Minecraft you want to install Fabric to, at least once. Otherwise, the installer will not be able to find the version's files.**
```

:::

> Once you've opened the installer, you should see a screen like this:

Reviewer 2: We could likely do with a section to aid MacOS users as there is one extra step. I can write it, just not sure where to put it?

Is it possible to have a collapsed section?

Author: Yeah, through the usage of the `details` block:

```md
::: details [Title Here]
This is a details block.
:::
```

Author: https://vitepress.dev/guide/markdown#custom-containers

![Fabric Installer with "Install" highlighted.](/assets/players/installing-fabric/installer-screen.png)

To install Fabric, simply choose your game version from the dropdown, and click `Install`.

> **You should make sure that 'Create Profile' is checked, otherwise the installer will not create a profile for you.**

Reviewer: ```suggestion
**Make sure that 'Create Profile' is checked.**
```

## 3. You're done!

> Once the installer has finished, you can open the Minecraft Launcher and select the Fabric profile from the dropdown in the bottom left corner and press play!

Reviewer: ```suggestion
Once the installer has finished, you can open the Minecraft Launcher and select the Fabric profile from the dropdown in the bottom left corner and press Play!
```

![Minecraft Launcher with Fabric profile selected.](/assets/players/installing-fabric/launcher-screen.png)

## Next Steps

Now that you've installed Fabric, you can add mods to your game! Check out the [Finding Mods](./finding-mods.md) guide for more information.

> If you encounter any issues whilst following this guide, you can ask for help in the [Fabric Discord](https://discord.gg/v6v4pMv) in the `#player-support` channel.

Reviewer: ```suggestion
If you encounter any issues while following this guide, you can ask for help in the [Fabric Discord](https://discord.gg/v6v4pMv) in the `#player-support` channel.
```
